static/app/utils/profiling/renderers/internalSampleTickRenderer.tsx

```diff
@@ -0,0 +1,66 @@
+import {mat3} from 'gl-matrix';
+
+import {colorChannelsToFillStyle} from '../colors/utils';
+import {Flamegraph} from '../flamegraph';
+import {FlamegraphTheme} from '../flamegraph/flamegraphTheme';
+import {getContext, Rect} from '../gl/utils';
+
+function computeIntervals(startedAt: number, weights: readonly number[]) {
```

Comment:
Nit: this is more like `computeAbsoluteSamples` or maybe `computeAbsoluteSampleTimestamps`